snakeware/external/configs/x86-64_defconfig

```diff
@@ -39,45 +494,3211 @@ BR2_PACKAGE_ALSA_UTILS_ASEQNET=y
+# BR2_PACKAGE_GENROMFS is not set
+# BR2_PACKAGE_IMX_USB_LOADER is not set
+# BR2_PACKAGE_MMC_UTILS is not set
+# BR2_PACKAGE_MTD is not set
+# BR2_PACKAGE_MTOOLS is not set
+# BR2_PACKAGE_NFS_UTILS is not set
+# BR2_PACKAGE_NILFS_UTILS is not set
+# BR2_PACKAGE_NTFS_3G is not set
+# BR2_PACKAGE_SP_OOPS_EXTRACT is not set
+# BR2_PACKAGE_SQUASHFS is not set
+# BR2_PACKAGE_SSHFS is not set
+# BR2_PACKAGE_UDFTOOLS is not set
+# BR2_PACKAGE_UNIONFS is not set
+# BR2_PACKAGE_XFSPROGS is not set
+
+#
+# Fonts, cursors, icons, sounds and themes
+#
+
+#
+# Cursors
+#
+# BR2_PACKAGE_COMIX_CURSORS is not set
+# BR2_PACKAGE_OBSIDIAN_CURSORS is not set
+
+#
+# Fonts
+#
+# BR2_PACKAGE_BITSTREAM_VERA is not set
+# BR2_PACKAGE_CANTARELL is not set
+# BR2_PACKAGE_DEJAVU is not set
+# BR2_PACKAGE_FONT_AWESOME is not set
+# BR2_PACKAGE_GHOSTSCRIPT_FONTS is not set
+# BR2_PACKAGE_INCONSOLATA is not set
+# BR2_PACKAGE_LIBERATION is not set
+
+#
+# Icons
+#
+# BR2_PACKAGE_GOOGLE_MATERIAL_DESIGN_ICONS is not set
+# BR2_PACKAGE_HICOLOR_ICON_THEME is not set
+
+#
+# Sounds
+#
+# BR2_PACKAGE_SOUND_THEME_BOREALIS is not set
+# BR2_PACKAGE_SOUND_THEME_FREEDESKTOP is not set
+
+#
+# Themes
+#
+
+#
+# Games
+#
+# BR2_PACKAGE_ASCII_INVADERS is not set
+# BR2_PACKAGE_CHOCOLATE_DOOM is not set
+# BR2_PACKAGE_FLARE_ENGINE is not set
+# BR2_PACKAGE_GNUCHESS is not set
+# BR2_PACKAGE_LBREAKOUT2 is not set
+# BR2_PACKAGE_LTRIS is not set
+
+#
+# minetest needs X11 and an OpenGL provider
+#
+# BR2_PACKAGE_OPENTYRIAN is not set
+# BR2_PACKAGE_PRBOOM is not set
+# BR2_PACKAGE_SL is not set
+
+#
+# solarus needs OpenGL and a toolchain w/ C++, gcc >= 4.8, NPTL, dynamic library
+#
+# BR2_PACKAGE_STELLA is not set
+
+#
+# Graphic libraries and applications (graphic/text)
+#
+
+#
+# Graphic applications
+#
+
+#
+# cog needs wpewebkit and a toolchain w/ threads
+#
+# BR2_PACKAGE_FSWEBCAM is not set
+# BR2_PACKAGE_GHOSTSCRIPT is not set
+
+#
+# glmark2 needs an OpenGL or an openGL ES and EGL backend
+#
+
+#
+# glslsandbox-player needs a toolchain w/ threads and an openGL ES and EGL driver
+#
+# BR2_PACKAGE_GNUPLOT is not set
+# BR2_PACKAGE_JHEAD is not set
+# BR2_PACKAGE_LIBVA_UTILS is not set
+BR2_PACKAGE_NETSURF_ARCH_SUPPORTS=y
+# BR2_PACKAGE_NETSURF is not set
+# BR2_PACKAGE_PNGQUANT is not set
+# BR2_PACKAGE_RRDTOOL is not set
+
+#
+# stellarium needs Qt5 and an OpenGL provider
+#
+# BR2_PACKAGE_TESSERACT_OCR is not set
+
+#
+# Graphic libraries
+#
+# BR2_PACKAGE_CEGUI06 is not set
+# BR2_PACKAGE_DIRECTFB is not set
+# BR2_PACKAGE_FB_TEST_APP is not set
+# BR2_PACKAGE_FBDUMP is not set
+# BR2_PACKAGE_FBGRAB is not set
+# BR2_PACKAGE_FBTERM is not set
+# BR2_PACKAGE_FBV is not set
+# BR2_PACKAGE_FREERDP is not set
+# BR2_PACKAGE_IMAGEMAGICK is not set
+# BR2_PACKAGE_LINUX_FUSION is not set
+# BR2_PACKAGE_MESA3D is not set
+# BR2_PACKAGE_OCRAD is not set
+
+#
+# ogre needs X11 and an OpenGL provider
+#
+# BR2_PACKAGE_PSPLASH is not set
+# BR2_PACKAGE_SDL is not set
```

Comment:
Isn't it a problem? I thought SDL is required for buildroot's audio functionality and there is no `BR2_PACKAGE_SDL_SOUND=1`